src/applications/vaos/referral-appointments/redux/selectors.js

```diff
@@ -9,3 +9,17 @@ export function getProviderInfo(state) {
     providerFetchStatus: state.referral.providerFetchStatus,
   };
 }
+
+export function getReferrals(state) {
+  return {
+    referrals: state.referral.referrals,
+    referralsFetchStatus: state.referral.referralsFetchStatus,
+  };
+}
+
+export function getReferral(state) {
+  return {
+    referrals: state.referral.referrals,
+    referralFetchStatus: state.referral.referralFetchStatus,
+  };
+}
```

Comment:
Is there a difference between these two functions besides the naming being plural and singular in the other? Seems they do the same thing.

Reply:
they are essentially the same and do close to the same thing. They both select referrals from the store but each sends back a different fetch status from the store, which when deployed will be for different API requests.

Reply:
No problem, that caught my eye too when I was going back over everything